Taxopark/DependencyInjection/DependencyContainer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Taxopark.Infrastructure.Data;
+
+namespace DependencyInjection
+{
+    public static class DependencyContainer
+    {
+        public static void AddDependencies(IServiceCollection services, IConfiguration configuration)
+        {
+            ConfigureMySQLContext(services, configuration);
```

Comment:
The more generic name (ex. ConfigureDbContext) can be used when you don't expect to use several databases.

Reply:
Good point, the method name was changed to ConfigureDBContext.